tools/scripts/features/frcnn/ImageVisualizer.ipynb

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
We usually land notebooks in `notebooks` branch and do not land any ipython code in master.

Reply:
Ah thats great to know thank you, I will adjust